src/plugins/home/public/application/components/usecase_overview/search_use_case_app.tsx

```diff
@@ -48,8 +70,83 @@ export const SearchUseCaseOverviewApp = ({ contentManagement }: Props) => {
     chrome.setBreadcrumbs(breadcrumbs);
   }, [chrome, isSearchUseCase]);
 
+  const dismissGetStartCards = async (state: boolean) => {
+    uiSettings.set(SEARCH_WORKSPACE_DISMISS_GET_STARTED, state);
+    setIsGetStartedDismissed(state);
+  };
+
+  useEffect(() => {
+    if (isGetStartedDismissed) {
+      page?.removeSection(SECTIONS.GET_STARTED);
+    } else {
+      page?.createSection(getStartedSection);
+    }
+  }, [isGetStartedDismissed, page]);
+
+  const hide = i18n.translate('home.searchOverview.getStartedCard.setting.hide.label', {
```

Comment:
Nit: since `show` is just text, maybe name it like `hideGetStartedLabel` sound more appropriate?

Reply:
make sense, i will update the name in follow up PR.